npm/ng-packs/packages/core/src/lib/services/timeout-limited-oauth.service.ts

```diff
@@ -1,8 +1,45 @@
-import { Injectable } from '@angular/core';
-import { OAuthService } from 'angular-oauth2-oidc';
+import { Inject, Injectable, NgZone, Optional } from '@angular/core';
+import {
+  AuthConfig,
+  DateTimeProvider,
+  HashHandler,
+  OAuthLogger,
+  OAuthService,
+  OAuthStorage,
+  UrlHelperService,
+  ValidationHandler,
+} from 'angular-oauth2-oidc';
+import { HttpClient } from '@angular/common/http';
+import { DOCUMENT } from '@angular/common';
 
 @Injectable()
 export class TimeoutLimitedOAuthService extends OAuthService {
+  constructor(
```

Comment:
I think the constructor can be removed

Reply:
I do not understand why but I got an error about the constructor. I have created the constructor and passed all things to super. It was fixed. But now, I removed it, and still t works.